### PR TITLE
feature/player_info

### PR DESCRIPTION
**Pull request by kerenvr**
• asks for amount of players
• asks user to confirm if number of players is correct
• gives each player a value (player1, player2, etc.)
• asks each player for their name
• creates player object based on name
• function that prints out score (will be used with the real score tracker)

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -243,21 +243,88 @@ def printHands(self, players: dict):
...
-    self.players = players
+    self.players = []
     self.value_wanted = int
     self.suit_wanted = ""
     self.player_asked = ""
     self.player_turn = ""
     self.info_player_turn = int
     self.info_player_asked = int
 
+  def multiplayer_info(self):
```
Comment (andredavii) [inline source/CardGames.py:256]: Code looks good! Appreciate the comments they helped me understand your approach more clearly.

Comment (kerenvr) [reply source/CardGames.py:256]: Thanks Andre, glad they helped! I'll continue to add comments so that it's easier to understand.

**Review by psk44 — APPROVED** [review]
Well-organized and easy-to-understand code! Thank you for leaving lots of comments!

**Comment by kerenvr** [comment]
> Well-organized and easy-to-understand code! Thank you for leaving lots of comments!

Thanks Peter! I'll continue to comment on my code!

**Review by Zwilson1301 — APPROVED** [review]
wow looks good Karen, i love the way that you made it so only the desired input will go through

**Comment by kerenvr** [comment]
> wow looks good Keren, i love the way that you made it so only the desired input will go through

Thanks Zack! Really appreciate it.

**Pull request merged**